client/README.md

```diff
@@ -86,6 +86,19 @@ The functional areas are in `packages` as noted below. Mostly the packages are s
 - host: the wrapper for the application which then hosts individual packages. Components in here include the framework - navigation, footer, drawer, app bar - and pages such as login
 - system: packages which are re-used by other packages, such as `item` and `name` but are application-specific, which is why they are in here and not in `common` which is trying to be more application agnostic.
 
+Packages should have the following hierarchy:
+
+```mermaid
+graph RL;
+    common --> config
+    system --> common
+    others["invoices, inventory, dashboard, etc"] --> system
+    host-->others;
+```
+
+Packages shouldn't have circular dependencies to other packages, i.e. packages can only import packages on their left (as shown in the diagram).
+For example, the `common` packages can be imported by the `system` package but the `common` package can't import the `system` or the `invoices` package.
+
```

Comment:
What about the "others" -- is it okay for them to import from each other?

Reply:
Good question, I would say yes its ok as long as they don't create circular dependencies.